cirq-core/cirq/contrib/noise_models/noise_models.py

```diff
@@ -54,7 +66,15 @@ def noisy_moment(self, moment: cirq.Moment, system_qubits: Sequence[cirq.Qid]):
         ]
         return output[::-1] if self._prepend else output
 
+    def _json_dict_(self) -> dict[str, object]:
+        return {'depol_prob': self.depol_prob, 'prepend': self._prepend}
+
+    @classmethod
+    def _from_json_dict_(cls, depol_prob, prepend, **kwargs):
+        return cls(depol_prob, prepend=prepend)
```

Comment:
When `_json_dict_` provides all `__init__` arguments, the `_from_json_dict_` method is redundant.  Please remove here and in other similar instances in this PR.

```suggestion
```

Reply:
Done in 7b00e93d415ab51018ec532dbaf5582f22c72ce9